eslint-plugin/docs/rules/image-src-attribute-not-empty.md

```diff
@@ -0,0 +1,26 @@
+# Disallow usage of image with empty source attribute (`@ecocode/image-src-attribute-not-empty`)
```

Comment:
I suggest to follow conventions and start the rule name by `no-`. But it also impacts name of your files..
```suggestion
# Disallow usage of image with empty source attribute (`@ecocode/no-empty-image-src-attribute`)
```